app/namespace.go

```diff
@@ -588,6 +612,30 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				return c.createNamespace(n, unp)
 			},
 		},
+		{
+			Name:    "add-region",
```

Comment:
imo there should be different endpoints for adding a region versus globalization. It should be intentional when a customer converts from a single region to multi-region namespace